kitchen.dokken.yml

```diff
@@ -33,3 +33,42 @@ platforms:
       pid_one_command: /bin/systemd
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
+
+  - name: amazonlinux-2
+    driver:
+      image: dokken/amazonlinux-2
+      pid_one_command: /usr/lib/systemd/systemd
+
+  - name: archlinux
+    driver:
+      image: archlinux
+      pid_one_command: /usr/lib/systemd/systemd
+
+  - name: gentoo
+    driver:
+      image: gentoo/stage3-amd64
+      pid_one_command: /sbin/init
+      intermediate_instructions:
+        - RUN echo 'GENTOO_MIRRORS=http://gentoo.mirrors.tera-byte.com/' >> /etc/portage/make.conf
+        - RUN mkdir -p /etc/portage/repos.conf
+        - RUN cp /usr/share/portage/config/repos.conf /etc/portage/repos.conf/gentoo.conf
+        - RUN sed -i -e 's/sync-uri =.*/sync-uri = rsync:\/\/rsync.us.gentoo.org\/gentoo-portage/' /etc/portage/repos.conf/gentoo.conf
+        - RUN emerge --sync || true
+
+suites:
```

Comment:
Looking at the other repos it looks like suites is only part of the kitchen.yml and not the dokken one. i think the suites for dokken are specified somewhere else and not in the kitchen file.
EX:
https://github.com/sous-chefs/apache2/blob/master/.github/workflows/ci.yml#L50
https://github.com/sous-chefs/apache2/blob/master/kitchen.dokken.yml

Reply:
Here suites are necessary as run_list for the tests can only be specified here.
Having suites ci.yml is to parallelize tests by CI matrix feature.

Reply:
haven't seen it in any of the other repos but i'll trust you on this one :) as long as the tests pass i am happy